ui/admin/tests/e2e/README.md

```diff
@@ -0,0 +1,49 @@
+# Admin UI e2e tests
+
+This test suite tests the Boundary Admin UI in an end-to-end setting. It was designed to be run in a
+variety of environments as long as the appropriate environment variables are set. You can choose to
+spin up your own infrastructure or use [Enos](https://github.com/hashicorp/boundary/tree/main/enos)
+to generate it for you.
+
+The test suite uses the [Playwright](https://playwright.dev/) framework.
+
+## Getting Started
+Setup...
+```bash
+cd boundary-ui/ui/admin
+yarn install
+npx playwright install # this installs the browsers used by Playwright
+```
+
+Ensure the `boundary` CLI is available on the path. You can validate this by making sure this
+command returns something.
+```bash
+which boundary
+```
+
+Run tests...
+```bash
+export BOUNDARY_ADDR=
+export E2E_PASSWORD_ADMIN_LOGIN_NAME=
+export E2E_PASSWORD_ADMIN_LOGIN_PASSWORD=
```

Comment:
I think I found a misspelling: I set the documented env variables, run the test and got a complain saying I do not have set `E2E_PASSWORD_ADMIN_PASSWORD` but the one we documented is `E2E_PASSWORD_ADMIN_LOGIN_PASSWORD`

Reply:
Also, as soon I run the test it complains I am missing: `E2E_TARGET_IP,E2E_SSH_USER,E2E_SSH_KEY_PATH,E2E_SSH_PORT` env variables.

Do you think we should document those as well?

Reply:
Yup. There's a typo in README. We need to set`E2E_PASSWORD_ADMIN_PASSWORD`. I'll update the README. 

So those are the extra "test-specific" environment variables that I was referencing earlier. Those are defined in each test file. You can see it near the top of `target.spec.js`

Reply:
Gotcha!! now I see them on `target.spec.js` 😉